schemas/src/registry_schemas/schemas/annual_report.json

```diff
@@ -13,9 +13,7 @@
             "required": [
                 "annualGeneralMeetingDate",
                 "annualReportDate",
-                "directors",
-                "mailingAddress",
-                "deliveryAddress"
+                "directors"
```

Comment:
I can't comment on this one, i will have to direct this one to @peter-freshworks.

Reply:
A property doesn't have to be in the required array to be in the schema. The definitions remain in the schema because if they are presented we do want to validate them. I have a ticket created to add offices to the annual report schema, but not having it in the schema presently doesn't break anything per se. Everything continues to work for co-op and we don't get a failure for not having address/agm date on BCORP. With that said - yes it is on the radar

Reply:
Thanks Peter. Please link the new ticket # here (the ticket to update the schema for registeredAddress/recordsAddress and also to make directors optional for BCorps).

Reply:
The ticket to fix the AR schema is /bcgov/entity#2014 and it is in the Backlog (ie, future).